h2/src/test/org/h2/test/store/TestMVStoreBenchmark.java

```diff
@@ -124,14 +124,6 @@ private static void addEntries(List<Map<Integer, String>> mapList, int size) {
     }
 
     static long getMemory() {
-        try {
-            LinkedList<byte[]> list = new LinkedList<>();
-            while (true) {
-                list.add(new byte[1024]);
-            }
-        } catch (OutOfMemoryError e) {
-            // ok
-        }
         for (int i = 0; i < 16; i++) {
```

Comment:
why remove this, not part of travis?

Reply:
Because IMHO OutOfMemoryError is never "ok", especially if it is triggered intentionally, just for the memory measurement purpose. Accuracy of that measurement is questionable anyway. Does getting OOME mean that there is absolutely no more memory available? Or smart JVM/GC will leave some of it aside? Will this gc()/wait() sequence ensure that all garbage is collected? Does Runtime.freeMemory() report exact number? So why bother, if price to pay is so high?